ospf6d/ospf6_asbr.c

```diff
@@ -1770,7 +1770,7 @@ int ospf6_redistribute_config_write(struct vty *vty, struct ospf6 *ospf6)
 		vty_out(vty, " redistribute %s", ZROUTE_NAME(type));
 		if (red->dmetric.value >= 0)
 			vty_out(vty, " metric %d", red->dmetric.value);
-		if (red->dmetric.type != DEFAULT_METRIC_TYPE)
+		if (red->dmetric.type == 1)
```

Comment:
should this be DEFAULT_DEFAULT_ALWAYS_METRIC?

Reply:
That constant refers to the metric of the default route created by the "default-information originate always" command. ospf6d doesn't seem to have constants to represent external route types (E1/E2) like ospfd does. Hopefully something that can be addressed on a separate PR later.

Reply:
Could/should we use the same metric constants for v2 and v3? Is there a reason we cannot, or reasons we should?

Reply:
@riw777 that would require a common header for the OSPF daemons in libfrr. I think the benefits would be minimal, since ospfd and ospf6d are largely different (they don't share a common base). I empathize with the idea of reducing code duplication, but sharing a few constants wouldn't make much of a difference in my opinion.